Code/Framework/AzCore/AzCore/Serialization/Utils.h

```diff
@@ -87,6 +88,39 @@ namespace AZ
         {
             return LoadObjectFromFileInPlace(filePath, AzTypeInfo<ObjectType>::Uuid(), &destination, context, filterDesc);
         }
+
```

Comment:
I am very aware that adding this function, and especially the above include of EditContext.h might be unpopular. I did this because it "Utils.h" does seem like the best place that I can find for this common function to be used by the DPE and the InstanceDataHierarchy. I am happily accepting suggestions of better placement if this in unpalatable to anyone!